src/main/java/hudson/plugins/git/util/BuildData.java

```diff
@@ -84,7 +98,25 @@ public String getIconFileName() {
     }
 
     public String getUrlName() {
-        return "git";
+        return index == null ? "git" : "git-"+index;
```

Comment:
I have not found usages, which may be impacted by the logic.
But it worth documenting potential behavior change in the Javadoc.